types/src/parcel/error.rs

```diff
@@ -101,6 +104,9 @@ impl Display for Error {
             Error::InvalidSignature(err) => format!("Parcel has invalid signature: {}.", err),
             Error::InconsistentShardOutcomes => "Shard outcomes are inconsistent".to_string(),
             Error::ParcelsTooBig => "Parcel size exceeded the body size limit".to_string(),
+            Error::RegularKeyAlreadyUsed => "The regular key is already registered to another account".to_string(),
+            Error::RegularKeyAlreadyUsedAsMaster => "The regular key is already used as the master account".to_string(),
+            Error::InvalidTransferReceiver => "Transfer receiver is not valid account".to_string(),
```

Comment:
What's the purpose of this error code?

Reply:
Transfer destination address is used as another account's regular key.